tests/allman/issue0256.d.ref

```diff
@@ -0,0 +1,11 @@
+void main()
+{
+    S s1 = {a: 3};
+    S s2 = {a: 3, b: "test string"};
+    S s3 = {a: 3, b: "test string", c: {x: 3.14, y: 3 + 4}};
+    T t = {
+        someStructMember1: 2, someStructMember2: 42, someStructMember3: null, // foobar
+        someOtherMember1: objA, someOtherMember2: objB, someOtherMember3: 0,
+        somethingMore: null, someFlagInThisStruct: -1
+    };
+}
```

Comment:
Shouldn't this be:
```D
void main()
{
    S s1 = {a: 3};
    S s2 = {a: 3, b: "test string"};
    S s3 = {a: 3, b: "test string", c: {x: 3.14, y: 3 + 4}};
    T t =
    {
        someStructMember1: 2,
        someStructMember2: 42,
        someStructMember3: null, // foobar
        someOtherMember1: objA,
        someOtherMember2: objB,
        someOtherMember3: 0,
        somethingMore: null,
        someFlagInThisStruct: -1
    };
}
```

1. Struct initializer brace on a new line
2. User newlines and indentation preserved

Reply:
for now it's just an improvement over what we had before (completely broken indentation), so I don't think we quite need to implement the full formatting fix with this PR